### PR TITLE
Add Riot.js

### PR DESCRIPTION
**Pull request by GianlucaGuarini**
I have added Riot.js to the party. Riot.js passes also the tests that were commented out on the other frameworks. Please let me know if I need to comment them out as well. 

Thank you

Screenshot 2019-08-31 at 11 01 54

### CODE REVIEW

**Comment by GianlucaGuarini** [comment]
@googlebot I signed it!

**Comment by GianlucaGuarini** [comment]
The CI fails for settings that are not related to this PR. Can I help you on that somehow?

**Comment by robdodson** [comment]
Hey @GianlucaGuarini, I'll try to take a look this week. Thanks for putting this together :)

@justinbmeyer—per our conversation in #580, I was curious if you or a member of your team might have time to also do a review of the tests?

File: libraries/riot/meta/summary.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<h4 id="riot-handling-data">Handling data</h4>
+
+By default, Riot.js passes all data to Custom Elements as attributes. If any of the attributes is not a primitive Riot.js passes it down to the Custom Element as DOM node property <code>domNode.objectAttribute = anObjectAttribute</code>
```
Comment (robdodson) [inline libraries/riot/meta/summary.md]:
```suggestion
By default, Riot.js passes all primitive data (strings, numbers, booleans) to Custom Elements as attributes. It passes complex data (Objects, Arrays) to Custom Elements as properties.
```

Tried to rephrase this a bit to simplify it.

**Comment by justinbmeyer** [comment]
I’ll talk to folks Monday.

File: libraries/riot/meta/summary.md (hunk 1 above)
Comment (GianlucaGuarini) [reply libraries/riot/meta/summary.md]: Thank you it makes sense.

**Comment by GianlucaGuarini** [comment]
@robdodson please let me know if I could do anything else to help you guys. Side Note: Riot.js could output custom components as well https://github.com/riot/custom-elements

**Comment by robdodson** [comment]
@GianlucaGuarini I noticed it's showing 26 basic and 14 advanced tests passed. But I think we only have 16 basic tests (that's what the other libraries show).

**Comment by GianlucaGuarini** [comment]
@robdodson that's what I meant with:

> Riot.js passes also the tests that were commented out on the other frameworks

Should I re-comment them out even if Riot.js could handle more unit tests without problems?

**Comment by robdodson** [comment]
oh! yes please comment them out. thank you!

**Comment by GianlucaGuarini** [comment]
@robdodson I have commented out the Riot.js tests, I think this decision might bias a bit the final results but at least it's consistent with the other frameworks.

**Comment by robdodson** [comment]
Rad thank you!

**Pull request merged**